src/components/SyncStatus/index.tsx

```diff
@@ -41,7 +41,7 @@ const SyncStatus = observer((props: Props) => {
             return (
                 <Icon
                     path={mdiCloseCircle}
-                    color="var(--ifm-color-danger)"
+                    color="var(--sync-status-error-color)"
```

Comment:
maybe use `mdiCloudCancelOutline` here?

Or should we even use the Cloud-symbol? It might not make sense anymore, when we start supporting a local api that stores to localhost?

Reply:
Hmm, the offline API is a good point... I'm actually not 100% happy with the cloud icon anyway. Here's my rationale for selecting the icon:

- Checkmark: Not ideal, can be confused with "correct answer" (status quo).
- Cloud: Makes sense for us (without considering the offline API), but from the student's point of view, their conceptual action is probably "save", rather than "upload".
- Floppy disk: The quintessential save icon. Makes the most sense semantically – but do younger users still recognize it?

I am leaning towards the last option though, since that icon is still quite common. It's relatively small in our case, but I think the key factor is to make sure that it's not mistaken as a "correct answer" sign.